x-pack/test/screenshot_creation/apps/ml_docs/transforms/transform_alerts.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FtrProviderContext } from '../../../ftr_provider_context';
+
+export default function ({ getPageObjects, getService }: FtrProviderContext) {
+  const transform = getService('transform');
+  const commonScreenshots = getService('commonScreenshots');
+  const screenshotDirectories = ['ml_docs', 'transforms'];
+  const pageObjects = getPageObjects(['triggersActionsUI']);
+  const testSubjects = getService('testSubjects');
+  const browser = getService('browser');
+  const actions = getService('actions');
```

Comment:
Just a nit, but it's easier to read if we group these things, so e.g. first have all page objects, then all service and then other variable definitions (I don't really care about the order of these "blocks", but having "blocks" at all helps IMO).

Reply:
Makes sense! Fixed in 46dee152e1970f5cd327d7138b6a3f8d69bf2044